### DISCUSSION TITLE
Error questions

### DISCUSSION THREAD

**Discussion by LParkhill** [NREL/REopt-API-Analysis#40] [1 - General REopt Questions] [answered]
### Error questions

**Comment by jisenergy** [comment]
Hello,
we've been getting the following error since yesterday. Is there a server maintenance going on? If so, when can we ping the server again? But if not, what could we possibly be doing wrong?

{
    "error_message": "Sorry, this request could not be processed. Please try again later."
}

Thanks so much,
JIS energy LLC

**Reply by LParkhill** [reply]
Hi JIS Energy,
We are working through some unanticipated problems related to Friday's update. It isn't a server maintenance issue and seems to only be impacting some evaluations.
Will let you know when the issues are resolved. 
We apologize for the inconvenience!!

**Reply by jisenergy** [reply]
Thanks a lot!

**Reply by LParkhill** [reply]
The developers believe that they have this issue fixed. Please let us know if you see further problems. Thanks for your patience!

**Reply by jisenergy** [reply]
Hi LParkhill,

We are just getting back into the last API job we posted and we are still getting the same error message:

{
"error_message": "Sorry, this request could not be processed. Please try again later."
}

This is a 15 minutes analysis job with CHP, PV, BESS, HOTTES and Chilled Water. Is that why?

**Reply by LParkhill** [reply]
So sorry that this isn't fixed for you. The developers asked if you could please send a run_uuid so that they can dig into it. Thanks!

**Reply by jisenergy** [reply]
Hi LParkhill,

It seems like the API wouldn't take 15 minute interval jobs. I never got a run uuid.
I tried a different run and it went through but it keeps giving me infeasible response. This is the run uuid for this job:
6f998d52-4221-420a-bb13-dcbcc637816e

Can you tell me what I am doing wrong?

**Reply by NLaws** [reply]
@jisenergy I believe that your problem is infeasible due to a combination of the values for:
- `renewable_electricity_min_pct`
- `co2_emissions_reduction_min_pct`
- `sr_required_pct`

I was able to get your scenario to solve by removing the first two constraints (i.e. letting them default to zero) and lowering the `sr_required_pct`.

**Comment by jisenergy** [comment, marked as answer]
Hi Nick,

 

Thank you so much for looking into this. I have another call that return bad input issue. I couldn’t figure out what’s wrong with my input values, would you be able to help me?

 

Run uuid:

18b95fb6-a7bf-476f-b716-a10174ca84ff

 

 


 <https://www.jisenergy.com/> 


Support


JIS energy LLC


 


P +19784735959


E ***@***.***


 


111 Middlesex Turnpike, #1006, Burlington, MA 01803


 <http://www.jisenergy.com/> www.jisenergy.com


 <https://www.facebook.com/jisenergy/> 


 <https://www.youtube.com/channel/UCMa_BGosQGh4uj0bWMKJX5g> 


 


 <https://www.linkedin.com/company/65327378/> 


 <https://www.jisenergy.com/products> 


The content of this email is confidential and intended for the recipient specified in message only. It is strictly forbidden to share any part of this message with any third party, without a written consent of the sender. If you received this message by mistake, please reply to this message and follow with its deletion, so that we can ensure such a mistake does not occur in the future.
